src/javascript/app_2/pages/trading/components/elements/account_switcher.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classNames from 'classnames';
+import { localize } from '../../../../../_common/localize';
+
+class AccountSwitcher extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            is_collapsed  : false,
+            active_account: this.props.active_account[0],
+        };
+    }
+
+    toggleAccountsList = () => {
+        this.setState({
+            is_collapsed: !this.state.is_collapsed,
+        });
+    }
+
+    switchAccount = (account) => {
+        this.setState({
+            active_account: account,
+        });
+        if (account.id !== this.state.active_account.id) {
+            if (this.props.onChange) {
+                this.props.onChange({ target: { name: 'currency', value: account.account_type } });
+            }
+        }
+    }
+
+    render() {
+        const account_list_collapsed = {
+            visibility: `${this.state.is_collapsed ? 'visible' : 'hidden'}`,
+        };
+
+        const account_is_hidden = (account) => (account.id === this.state.active_account.id);
```

Comment:
Would be easier to distinguish between variable names and function names if it follows the naming conventions. (camelCase for functions)

And since it returns a boolean value, better to have the `is` at the beginning: `isAccountHidden`

Reply:
Sure, will do the changes.